core/domain/suggestion_domain.py

```diff
@@ -14,8 +14,15 @@
 
 """Domain object for Oppia suggestions."""
```

Comment:
Weren't we planning to rename this to suggestion_registry.py?

I'm open either way, though; feel free to decide based on what you think is most intuitive.

Reply:
Oh I missed this last commit.. Will rename it as it feels better that way!

Reply:
Done :)